app/assets/stylesheets/customsdecexfrontend-app.scss

```diff
@@ -19,5 +19,10 @@
 $govuk-assets-path: '/customs-declare-exports/assets/';
 $govuk-images-path: "/customs-declare-exports/assets/lib/govuk-frontend/govuk/assets/images/";
 $govuk-fonts-path: "/customs-declare-exports/assets/lib/govuk-frontend/govuk/assets/fonts/";
+
 @import 'lib/govuk-frontend/govuk/all';
+@import 'lib/accessible-autocomplete/dist/accessible-autocomplete.min';
```

Comment:
Do these imports have to be below govuk/all?   Someone asked me that question recently

Reply:
No it doesn't